source/blog/_sidebar.html.erb

```diff
@@ -65,13 +65,12 @@
   <h3 class="list-group-title" id="browse-subject">Browse by Subject</h3>
   <ul class="list-group" aria-labelledby="browse-subject">
     <li class="list-item"><a href="<%= tag_path 'Announcement'%>">Announcements</a></li>
-    <li class="list-item"><a href="<%= tag_path 'Ember CLI'%>">Ember CLI</a></li>
+    <li class="list-item"><a href="<%= tag_path 'Community'%>">Community</a></li>
     <li class="list-item"><a href="<%= tag_path 'Ember Data'%>">Ember Data</a></li>
-    <li class="list-item"><a href="<%= tag_path 'Performance'%>">Performance</a></li>
+    <li class="list-item"><a href="<%= tag_path 'Leadership'%>">Leadership</a></li>
     <li class="list-item"><a href="<%= tag_path 'Roadmap'%>">Roadmap</a></li>
     <li class="list-item"><a href="<%= tag_path 'Security'%>">Security</a></li>
+    <li class="list-item"><a href="<%= tag_path 'Tech'%>">Tech</a></li>
```

Comment:
I'm _assuming_ that most posts that fall under the category "Tech" are already covered by either the `Releases` or the `Ember CLI` / `Ember Data` tags. What are your thoughts?